packages/fxa-payments-server/src/lib/account.test.ts

```diff
@@ -44,6 +53,7 @@ describe('lib/account', () => {
       );
       expect(updateAPIClientToken).not.toHaveBeenCalled();
       expect(sentry.captureException).toHaveBeenCalledWith(FXA_SIGNUP_ERROR);
+      expect(ReactGALog.logEvent).not.toBeCalledWith(MOCK_EVENTS.SignUp);
```

Comment:
Q: Wasn't clear from https://jestjs.io/docs/expect#tohavebeencalledwitharg1-arg2-, but what does `.not.toBeCalledWith()` check for?
That the function was called +1&times;, but not with those specific args?
Or the function was not called at all? (although that feels like an odd way to test that; versus `.not.toBeCalled()`)

Looks like we only have one other usage of `.notToHaveBeenCalledWith()` currently:
```sh
git grep -Pn "\.not\.to(Be|HaveBeen)CalledWith\(" packages/fxa-payments-server/ | cat

packages/fxa-payments-server/server/lib/routes/legal-docs.test.js:89:    expect(mockFetch).not.toHaveBeenCalledWith(`${validUrl}.ka.pdf`, {
```

Reply:
Aligned that `.not.toBeCalledWith()` should include args and to use `.not.toBeCalled()` when there aren't any.